src/Adapters/EmbedPresenter/ParserFunctionEmbedPresenter.php

```diff
@@ -50,4 +54,9 @@ public function getParserFunctionReturnValue(): array {
 		];
 	}
 
+	public function loadRendererRequirements( ContentRenderer $renderer ): void {
```

Comment:
Is this the right place for loading presentation-related requirements? Otherwise I need to inject something like a "RendererRequirementsLoader` into `EmbedUseCase`.

Reply:
> The one concern with always loading the resources is it's over 500kb of JS/CSS.

Which happens only without reason when you use #embed to render markdown.